docs/manual/mkdocs.yml

```diff
@@ -70,6 +70,9 @@ nav:
       - schemaextension:
         - schemaextension add: 'cmd/graph/schemaextension/schemaextension-add.md'
         - schemaextension get: 'cmd/graph/schemaextension/schemaextension-get.md'
+    - OneDrive (onedrive):
+      - report:
+        - report onedriveactivityfilecounts: 'cmd/onedrive/report/report-onedriveactivityfilecounts.md'
```

Comment:
Following the naming conventions, the file name should be `report-activityfilecounts.md`